### PR TITLE
Advertise the Navigator as a development tool

### PR DESCRIPTION
**Pull request by stefanobaghino-da**
Rather than a generic inspection tool, which it is not designed to be.

Furthermore, deprecated customization and redirect readers to the getting started guide for an example of how to use `create-daml-app` for rapid prototyping.

changelog_begin
[Navigator] Custom views are deprecated, use `create-daml-app` instead for rapid prototyping
changelog_end

### CODE REVIEW

File: docs/source/tools/navigator/index.rst [hunk 1]
```diff
@@ -403,11 +397,11 @@ To debug config file errors and learn more about the
 config file API, open the Navigator ``/config`` page in your browser
 (e.g., `<http://localhost:7500/config>`_).
 
-Using Navigator with a Daml Ledger
-==================================
+Using Navigator with a secured sandbox
```
[the remark this answers is not in the document]
Comment (carrielaben-da) [reply docs/source/tools/navigator/index.rst:400]: This is correct, we'd want title case here.

File: docs/source/tools/navigator/index.rst [hunk 2]
```diff
@@ -4,9 +4,7 @@
 Navigator
 #########
 
-The Navigator is a front-end that you can use to connect to any Daml Ledger and inspect and modify the ledger. You can use it during Daml development to explore the flow and implications of the Daml models.
-
-The first sections of this guide cover use of the Navigator with the SDK. Refer to :ref:`navigator-manual-advanced-usage` for information on using Navigator outside the context of the SDK.
+The Navigator is a front-end development tool that you can use to inspect and modify the state of a sandbox. You can use it during Daml development to explore the flow and implications of the Daml models.
```
Comment (carrielaben-da) [inline docs/source/tools/navigator/index.rst:7]:
```suggestion
The Navigator is a front-end development tool used to inspect and modify the state of a sandbox. You can use it during Daml development to explore the flow and implications of the Daml models.
```

**Review by carrielaben-da — APPROVED** [review]

**Review by ray-roestenburg-da — APPROVED** [review]

File: docs/source/tools/navigator/index.rst [hunk 3]
```diff
@@ -330,11 +326,9 @@ Advanced Usage
 Customizable table views
 ========================
 
-Customizable table views is an advanced rapid-prototyping feature,
-intended for Daml developers who wish to customize the Navigator UI without
-developing a custom application.
-
-.. COMMENT: Suggest changing para below to procedure format.
+.. warning::
+    Customizable table views are deprecated. For rapid prototyping we recommend you use the skeleton ``create-daml-app``.
```
[the remark this answers is not in the document]
Comment (stefanobaghino-da) [reply docs/source/tools/navigator/index.rst:330]: That is the only example to the best of my understanding. I would:
1. merge as is
1. add one or two low priority tickets to the board to
    1. replace its usage in the `quickstart-java`
    1. if feasible, add a warning when using it

@ray-roestenburg-da thoughts?

[the remark this answers is not in the document]
Comment (stefanobaghino-da) [reply docs/source/tools/navigator/index.rst:330]: I'll let @ray-roestenburg-da take the final decision.

[the remark this answers is not in the document]
Comment (stefanobaghino-da) [reply docs/source/tools/navigator/index.rst:330]: @ray-roestenburg-da What do you think? How should we proceed?

[the remark this answers is not in the document]
Comment (RayRoestenburg) [reply docs/source/tools/navigator/index.rst:330]: Agree with Moritz, sounds good

[the remark this answers is not in the document]
Comment (stefanobaghino-da) [reply docs/source/tools/navigator/index.rst:330]: Ok, I'll close this and open a ticket then. Thanks!

[the remark this answers is not in the document]
Comment (stefanobaghino-da) [reply docs/source/tools/navigator/index.rst:330]: Created https://github.com/digital-asset/daml/issues/14234, closing.

**Comment by stefanobaghino-da** [comment]
Closed in favor of a larger fix that includes issuing a warning upon usage and removing the usage from `quickstart-java` (see https://github.com/digital-asset/daml/issues/14234).

**Comment by LeventeBarczyDA** [comment]
Can you distinguish what the differences are between a "generic inspection" tool and a "development tool"?

**Comment by stefanobaghino-da** [comment]
> Can you distinguish what the differences are between a "generic inspection" tool and a "development tool"?

Navigator was not designed to inspect a generic ledger at scale, since it works by keeping a local, running dump of the whole ledger with the assumption of total visibility over all parties. These are safe assumptions in a development setup but don't work to generally inspect any ledger with secured access and possibly holding a very high number of transactions.

**Pull request closed without merging**